docs/guides/mobile-plugin-gma-android.md

```diff
@@ -0,0 +1,58 @@
+# UID2 GMA Plugin for Android Integration Guide
+
+## Introduction
+
+The UID2 Google Mobile Ads (GMA) Plugin for Android enables publishers that use the [Google Mobile Ads (GMA) SDK](https://developers.google.com/ad-manager/mobile-ads-sdk) to include UID2 data in ad requests from Android apps. This is done automatically, with no direct coding needed from the app developer.
+
+## Functionality
+
+This plugin simplifies integration with Google Mobile Ads (GMA) for any publishers who want to support UID2 for apps running on Android devices.
+
+## Version
+
+<!-- As of 2023-07-15 -->
```

Comment:
should we remove these dates if we are not showing them?

Reply:
@sunnywu : I kept this info, commented, in case it might come in useful. I checked -- comments are not included in the output. Happy to remove if you don't think there is even any internal use.

Reply:
yeh there's no need and can remove - i added it assuming we would show it on the page.